.vscode/c_cpp_properties.json

```diff
@@ -3,7 +3,8 @@
         {
             "name": "Linux",
             "includePath": [
-                "${workspaceFolder}/include/**"
+                "${workspaceFolder}/include/**",
+                "/usr/include/python3.11"
```

Comment:
this seems very specific to your setup

Reply:
done, sorry didnt mean to include this in the commit